common/training/switchout.py

```diff
@@ -33,8 +33,8 @@ def switchout_target(self, source, targetb_blank_idx: int,
   def get_switched():
     x_ = x
     shape = tf.shape(x)
-    n_batch = tf.shape(x)[data.batch_dim_axis]
-    n_time = tf.shape(x)[data.time_dim_axis]
```

Comment:
Why changes in Switchout? This looks very unrelated to this PR. Please separate things.